gfop/get_food_counts.py

```diff
@@ -31,6 +31,32 @@ def get_sample_types(simple_complex='all'):
     return (gfop_metadata[['filename', *col_sample_types]]
             .set_index('filename'))
 
+def get_sample_metadata(gnps_network, all_groups):
+    '''
+    Args:
+        gnps_network(dataframe): Dataframe generated from classical molecular networking
+                                  with study dataset(s) and reference dataset.
+        groups(list): can contain 'G1', 'G2' to denote study spectrum files.
```

Comment:
**todo:** Make sure the arguments in the docstring match the actual variable names (`groups` vs `all_groups`).